packages/core/src/Page/index.js

```diff
@@ -443,6 +443,7 @@ class Page {
       headerIdMap: this.headerIdMap,
       ignore: this.pageConfig.ignore,
       addressablePagesSource: this.pageConfig.addressablePagesSource,
+      intrasiteLinkValidation: this.pageConfig.intrasiteLinkValidation,
```

Comment:
`LayoutManager` / `ExternalManager` (new classes) would need this too.

Reply:
Added to these classes! Thanks for pointing it out.